documentation/docs/20-core-concepts/20-load.md

```diff
@@ -450,6 +450,46 @@ export function load() {
 </script>
 ```
 
+## Defer loading slow data
```

Comment:
This section duplicates some parts of the section above ("Promise unwrapping") - maybe we should merge them?